client/my-sites/upgrades/domain-management/edit/registered-domain.jsx

```diff
@@ -87,7 +97,7 @@ const RegisteredDomain = React.createClass( {
 		return <DomainWarnings
 			domain={ this.props.domain }
 			selectedSite={ this.props.selectedSite }
-			ruleWhiteList={ [ 'expiredDomains', 'expiringDomains', 'newDomainsWithPrimary', 'newDomains' ] } />;
+			ruleWhiteList={ [ 'expiredDomains', 'expiringDomains', 'newDomainsWithPrimary', 'newDomains' ] }/>;
```

Comment:
I'm not a fan of introducing whitespace changes in the same commit, it's hard to see if it's related to the main change or not.
Unfortunately, I can't find any rules if a space should be here or not :laughing: But grepping (or rather `ag`ing) through our source code it seems that adding a space before the closing tag (the original version) in this case is the definitely more popular choice. So unless you want to introduce changes like this too often, I'd recommend configuring your editor accordingly :) Sorry for nitpicking like this, it's my pet peeve... :disappointed: